File: CoWorkHub/UI/coworkhub_mobile/lib/models/password_reset_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'password_reset_request.g.dart';

@JsonSerializable()
class PasswordResetRequest {
  int userId;
  String code;
  DateTime createdAt;

  PasswordResetRequest({
    required this.userId,
    required this.code,
    required this.createdAt,
  });

  factory PasswordResetRequest.fromJson(Map<String, dynamic> json) =>
      _$PasswordResetRequestFromJson(json);

  Map<String, dynamic> toJson() => _$PasswordResetRequestToJson(this);
}

File: CoWorkHub/CoWorkHub.Model/PasswordResetRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CoWorkHub.Model
{
    public class PasswordResetRequest
    {
        public int UserId { get; set; }
        public string Code { get; set; } = null!;
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}
